docs/riff_system_install.md

```diff
@@ -35,13 +35,6 @@ riff system install [flags]
       --node-port         whether to use NodePort instead of LoadBalancer for ingress gateways
 ```
 
-### Options inherited from parent commands
```

Comment:
should still support kubeconfig

Reply:
It actually doesn't (currently)

Reply:
that's a bug, we should pass these flags to kubectl when we shell out to it.